src/transformers/generation/configuration_utils.py

```diff
@@ -324,6 +324,10 @@ class GenerationConfig(PushToHubMixin):
               reduce by 1. `num_assistant_tokens` value is persistent over multiple generation calls with the same assistant model.
             - `"heuristic_transient"`: Same as `"heuristic"` but `num_assistant_tokens` is reset to its initial value after each generation call.
             - `"constant"`: `num_assistant_tokens` stays unchanged during generation
+        assistant_confidence_threshold (`float`, *optional*, defaults to 0):
```

Comment:
Feel free to add a reference to your paper in this docstring 🤗 It will be useful in the long run for everyone, to track the origin of the technique!